sdk/eventhubs/azure_messaging_eventhubs_checkpointstore_blob/README.md

```diff
@@ -58,15 +58,15 @@ cargo add azure_identity tokio azure_messaging_eventhubs azure_storage_blob
 
 Before interacting with Azure services, you need a credential to use to authenticate and authorize the user.
 
-The example shown below uses a [`DefaultAzureCredential`][default_cred_ref], which is appropriate for most local development environments. Additionally, we recommend using a managed identity for authentication in production environments. You can find more information on different ways of authenticating and their corresponding credential types in the [Azure Identity] documentation.
+The example shown below uses a [`DeveloperToolsCredential`][default_cred_ref], which is appropriate for most local development environments. Additionally, we recommend using a managed identity for authentication in production environments. You can find more information on different ways of authenticating and their corresponding credential types in the [Azure Identity] documentation.
```

Comment:
Should change the link name to avoid confusion e.g., `dev_tool_ref`.

Tip: if you call the link the same as the text, you don't need two brackets:

```markdown
Use the [`DeveloperToolCredential`] to ...

[`DeveloperToolCredential`]: https://contoso.com
```

Reply:
I figured I would do that once we go back to clean these up after Identity ships - this still maintains the DefaultAzureCredential link so it's searchable.